content/en/docs/concepts/services-networking/ingress.md

```diff
@@ -109,13 +110,101 @@ Each HTTP rule contains the following information:
 A default backend is often configured in an Ingress controller to service any requests that do not
 match a path in the spec.
 
-### Default Backend
+### Path Types
+
+Each path in an Ingress has a corresponding path type. There are three supported
+path types:
+
+* **ImplementationSpecific:** With this default path type, matching is up to the
```

Comment:
Since these strings are like values of an enum, I'd write them in backticks and make them italic not bold.
I haven't tried this so if you do and it looks awful, that is fair enough.

You could also write for this line:
```suggestion
* _`ImplementationSpecific`_ (default): With this path type, matching is up to the
```

If you do that, I'd move the colon characters for the other two path types to be outside the closing `_` (ie, not italic).

Reply:
Good call, should be updated now, thanks!